packages/graphql-transformers-e2e-tests/src/__tests__/ModelConnectionTransformer.e2e.test.ts

```diff
@@ -464,18 +526,81 @@ test('Test album self connection.', async () => {
                 title
             }
         }
-    }`, {})
-    expect(createSelfAlbum.data.createAlbum.id).toBeDefined()
-    expect(createSelfAlbum.data.createAlbum.title).toEqual('A Album!')
-    expect(createSelfAlbum.data.createAlbum.parent.id).toEqual(createAlbum.data.createAlbum.id)
-    expect(createSelfAlbum.data.createAlbum.parent.title).toEqual(createAlbum.data.createAlbum.title)
+    }`,
+    {}
+  );
+  expect(createSelfAlbum.data.createAlbum.id).toBeDefined();
+  expect(createSelfAlbum.data.createAlbum.title).toEqual('A Album!');
+  expect(createSelfAlbum.data.createAlbum.parent.id).toEqual(createAlbum.data.createAlbum.id);
+  expect(createSelfAlbum.data.createAlbum.parent.title).toEqual(createAlbum.data.createAlbum.title);
 
-    const queryAlbum = await GRAPHQL_CLIENT.query(`query {
+  const queryAlbum = await GRAPHQL_CLIENT.query(
+    `query {
         getAlbum(id: "${createAlbum.data.createAlbum.id}") {
             id
             title
         }
-    }`, {})
-    expect(queryAlbum.data.getAlbum).toBeDefined()
-    expect(queryAlbum.data.getAlbum.title).toEqual('Test Album')
-})
+    }`,
+    {}
+  );
+  expect(queryAlbum.data.getAlbum).toBeDefined();
+  expect(queryAlbum.data.getAlbum.title).toEqual('Test Album');
+});
+
+test('Test default limit is 50', async () => {
+  // create Auth logic around this
+  const postID = 'e2eConnectionPost';
+  const postTitle = 'samplePost';
+  const createPost = await GRAPHQL_CLIENT.query(
+    `mutation CreatePost {
+        createPost(input: {title: "${postTitle}", id: "${postID}"}) {
+        id
+          title
+        }
+      }
+      `,
+    {}
+  );
+  expect(createPost.data.createPost).toBeDefined();
+  expect(createPost.data.createPost.id).toEqual(postID);
+  expect(createPost.data.createPost.title).toEqual(postTitle);
+
+  for (let i = 0; i < 52; i++) {
```

Comment:
Doesn't this create 51 comments even though you're expecting 50?

Reply:
changed in the latest commit